src/pages/resultsView/ResultsViewPageStoreUtils.spec.ts

```diff
@@ -1707,6 +2316,74 @@ describe('getSampleAlteredMap', () => {
         assert.deepEqual(ret["RAS"], expectedResult["RAS"], "merged gene track(with group name)");
 
     });
+
+    it('should even handle not profiled samples correctly', () => {
```

Comment:
can you rewrite this to explain what it means for it to handle it correctly? i.e. "not profiled samples are excluded" or is it "not profiled samples are considered unaltered" ?

Reply:
@adamabeshouse How about "should set undefined for the not profiled samples"?

Reply:
perfect